odd_collector/adapters/superset/client.py

```diff
@@ -0,0 +1,213 @@
+from typing import List, Dict, Any, NamedTuple, Optional, Tuple, Union
+from urllib.parse import urlparse
+from .domain.column import Column
+from .domain.chart import Chart
+from .domain.dashboard import Dashboard
+from odd_collector.domain.plugin import SupersetPlugin
+from .domain.dataset import Dataset
+from json import dumps
+
+import asyncio
+import aiohttp
+
+
+class RequestArgs(NamedTuple):
+    method: str
+    url: str
+    params: Optional[Dict[Any, Any]] = None
+    headers: Optional[Dict[Any, Any]] = None
+    payload: Optional[Dict[Any, Any]] = None
+
+
+class SupersetClient:
```

Comment:
1) Place private methods under the public
2) Why did you do some methods private(__%) and another protected(_%), make them private with double underscore